src/modules/meta_producto/meta_producto.service.ts

```
@@ -1,673 +1,726 @@
-    Injectable,
-    BadRequestException,
-    InternalServerErrorException,
+  Injectable,
+  BadRequestException,
+  InternalServerErrorException,
 } from '@nestjs/common';
-import { MetaProductoRequest, MetaProductoResponse, MetaProducto } from './dto/meta_producto.dto';
+import {
+  MetaProductoRequest,
+  MetaProductoResponse,
+  MetaProducto,
```

ESLint

[error] 'MetaProducto' is defined but never used.

(@typescript-eslint/no-unused-vars)